### ISSUE TITLE
plot_metagene() normailzation

### ISSUE THREAD

**Issue by hakanozadam** [ribosomeprofiling/riboR_alpha#6]
Let's have an options, "normalize = False"
in the plot_metagene function.

When False, it plots the raw values (as it is doing now)

If normalize = True, then for each experiment we can normalize the metagene vector by the total number of reads (this is already calculated and stored in the ribo file).

For example,

normalized_metagene_vector = (metagene_vector / experiment_taotal_reads ) * 1000

Since normalization will give us very small values we can multiply by 1000 to get values larger than 1 one the y axis.

@cancenik , do you have other suggestions for normalizing metagene plots?

**Comment by hakanozadam** [comment]
Let's revisit this. I don't see "normalize" option in region counts.

**Comment by mjgeng** [comment]
I made a new parameter called 'normalize' as well. The default value is set to FALSE. 
Here are some examples.

The differences are a little more apparent on the green data because there are more experiments.

**normalize = FALSE:**
`green <- ribo("green_6_10_2019.ribo")`
`plot_metagene(green, "start", 15, 40, normalize = FALSE, tick = 10)`
![metagene_raw_counts](https://user-images.githubusercontent.com/15066495/60132161-968d2d80-9760-11e9-97c4-ba0b847bd002.png)


**normalize = TRUE:**
`plot_metagene(green, "start", 15, 40, normalize = TRUE, tick = 10)`
![metagene_normalized](https://user-images.githubusercontent.com/15066495/60130196-6479cc80-975c-11e9-97c0-58c5a93914e9.png)

**Notes**
The normalized value is computed as (metagene.count)/(experiment.total.reads) * 1000. The experiment.total.reads is the number of reads at for each  given count's respective experiment.

The sources of the total reads came from the 'experiments.info' element of the ribo object. This makes things a little more difficult in the case that the user passes in a data.table that corresponds to the output of  the original **get_tidy_metagene()**. When this occurs, there is no convenient way for me to get the total reads, as the function no longer has access to the ribo object.

There are a few options to combat this. I decided to change the **get_tidy_metagene()** function to add a new column for total.reads, so now, **get_tidy_metagene()** has the original 'experiment', 'position', and 'count' columns **with an added 'total.reads' column**. The one downside is that the added column may serve no use when normalize = FALSE. However, I feel that this option makes it as easy as possible on the user. The outputted data.table of **get_tidy_metagene()** will easily accomodate both 'normalize' conditions, and the maximum wasted space is a little over 2600 values (26 read lengths * 101 positions) , which is not a large amount of memory.
